app/api/probe_dock/categories_api.rb

```diff
@@ -37,16 +37,21 @@ def current_organization
 
         rel = Category
 
-        rel = paginated rel do |rel|
+        rel = paginated rel do |paginated_rel|
           if current_organization
             # filter by organization
-            rel = rel.where 'categories.organization_id = ?', current_organization.id
+            paginated_rel = paginated_rel.where 'categories.organization_id = ?', current_organization.id
```

Comment:
Add parentheses to method call (part of our ongoing refactoring to follow the new conventions). Also on lines 40 and 46.